proposals/0069_prometheus-exporters-otel-embedding.md

```diff
@@ -0,0 +1,286 @@
+
+Meanwhile, the Prometheus community has developed hundreds of exporters over many years, many of which are stable. Creating parallel implementations in the OpenTelemetry ecosystem wastes community resources and often results in "drive-by contributions" that are abandoned shortly after acceptance.
+
+### Pitfalls of the current solution
+
+1. **Duplication of Work**: Infrastructure monitoring receivers are reimplemented in OpenTelemetry when functionally equivalent Prometheus exporters already exist.
+
+2. **Unmaintained Components**: Many OpenTelemetry receivers that replicate Prometheus exporter functionality become unmaintained in early development stages.
+
+3. **Quality and Stability Issues**: The pressure to provide comprehensive coverage leads to accepting components that may not meet quality standards, contributing to collector-contrib's stability problems.
+
+4. **Diverging Ecosystems**: Two communities are solving the same problems independently, fragmenting effort and expertise.
+
+5. **Maintenance Burden**: Both ecosystems must independently maintain similar functionality for monitoring the same infrastructure components.
+
+## Goals
+
+* Enable embedding of Prometheus exporters as native OpenTelemetry Collector receivers via the OpenTelemetry Collector Builder (OCB).
+* Reduce duplication of effort between Prometheus and OpenTelemetry communities.
+* Maintain the "single binary" promise for users who want comprehensive telemetry collection.
+* Leverage existing, mature Prometheus exporters instead of reimplementing them in OTel Collector's side.
+* Unify the two ecosystems to increase the likelihood of attracting more maintainers and contributors.
+
+### Audience
+
+* Prometheus exporter maintainers and developers
+* OpenTelemetry Collector users and contributors
+* Organizations using both Prometheus and OpenTelemetry in their observability stack
+* Distribution builders (e.g., Grafana Alloy, OllyGarden Rose, AWS ADOT, DataDog DDOT, Elastic EDOT)
+
+## Non-Goals
+
+* Replace the existing OpenTelemetry Prometheus receiver (which scrapes Prometheus endpoints).
+* Force all Prometheus exporters to implement embedding interfaces immediately.
+* Automatically remove existing OpenTelemetry receivers that overlap with Prometheus exporters (this follows OpenTelemetry's own component removal policy).
+* Mandate that OpenTelemetry collector-contrib includes embedded Prometheus exporters (distributions can be customized).
+
+## How
+
+### Overview
+
+Prometheus exporters function similarly to OpenTelemetry Collector receivers: they gather information from infrastructure and expose it as metrics. The key difference is the output format and collection mechanism. Prometheus exporters expose an HTTP endpoint (typically `/metrics`) that is scraped, while OpenTelemetry receivers push metrics into a pipeline.
```

Comment:
And Prometheus exporters are only concerned with metrics, no other telemetry signals.